mappers/modbus-go/config.go

```diff
@@ -77,8 +77,7 @@ func (c *Config) parseFlags() error {
 	pflag.StringVar(&c.Mqtt.PrivateKey, "mqtt-priviatekey", c.Mqtt.PrivateKey, "private key file path")
 	pflag.Parse()
 
-	if (c.Mqtt.Cert != "" && c.Mqtt.PrivateKey == "") ||
-		(c.Mqtt.Cert == "" && c.Mqtt.PrivateKey != "") {
+	if c.Mqtt.Cert == "" || c.Mqtt.PrivateKey == "" {
```

Comment:
If c.Mqtt.Cert and c.Mqtt.PrivateKey are both empty, maybe we shouldn't return error? @sailorvii

Reply:
If it is like you say, it is better to change error message.

Reply:
We should support unsecure connection?

Reply:
For the Mqtt broker could be external, the authentication method depends on what the broker requires.
So I think it's better to support different methods.